airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/write/InputConsumer.kt

```diff
@@ -16,6 +16,10 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
 /** Runnable input consumer. */
+// What does the type parameter mean here?
+// Why isn't this a Flow instead?
+// It looks like you could have a root Flow<String> representing each line in the InputStream
+// to which you chain some mapper which deserializes, etc.
```

Comment:
That makes sense. It's more natural for the writer to aggregate the consumer than the other way around, and for the consumer to be type agnostic.